v0/destinations/zendesk/transform.js

```diff
@@ -347,15 +435,30 @@ async function processIdentify(message, destinationConfig, headers) {
   const url = endPoint + category.createOrUpdateUserEndpoint;
   const returnList = [];
 
-  const traits = getFieldValueFromMessage(message, "traits");
+  const userId = await getUserId(message, headers);
+
+  if (primaryEmail && userId) {
+    const userIdentityId = await getUserIdentityId(
+      userId,
+      primaryEmail,
+      headers,
+      destinationConfig
+    );
+    if (userIdentityId) {
+      returnList.push(
+        responseBuilderToSetPrimaryAccount(userIdentityId, userId, headers)
+      );
+    }
+    delete traits.primaryEmail;
```

Comment:
We are already deleting it in line no 419, right?

Reply:
Updated